src/utils/config/config.go

```diff
@@ -44,6 +43,9 @@ type Config struct {
 	// So this capacity should account for interactions that may appear during a few second window when the previous batch is inserted to the database.
 	ListenerQueueSize int
 
+	// Distance from the current chain height that is considered stable. Newer blocks won't be downloaded.
+	ListenerStableDistance int64
```

Comment:
I don't like this name.

Sth like. 'RequiredConfirmationBlocks' imho would be a better name.

with a comment:
```
// a minimum amount of confirmations (blocks on top of the given block) that are required to consider 
// a given block as confirmed (i.e. not being a fork)
```